src/nvim/options.lua

```diff
@@ -1599,7 +1599,7 @@ return {
       deny_duplicates=true,
       alloced=true,
       varname='p_nf',
-      defaults={if_true={vi="octal,hex", vim="hex"}}
+      defaults={if_true={vi="bin,octal,hex", vim="bin,hex"}}
```

Comment:
Bram did accept this vi default? Actually it is incorrect: vi default meant to be compatible with old vi and this is new functionality, so it by no means can appear there.